### ISSUE TITLE
build error for 3.0.0

### ISSUE THREAD

**Issue by smondet** [avsm/ocaml-dockerfile#6]
```
#=== ERROR while installing dockerfile.3.0.0 ==================================#
# opam-version         1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os                   linux
# command              jbuilder build -p dockerfile -j 4
# path                 /home/opam/.opam/4.03.0/build/dockerfile.3.0.0
# compiler             4.03.0
# exit-code            1
# env-file             /home/opam/.opam/4.03.0/build/dockerfile.3.0.0/dockerfile-7-b38251.env
# stdout-file          /home/opam/.opam/4.03.0/build/dockerfile.3.0.0/dockerfile-7-b38251.out
# stderr-file          /home/opam/.opam/4.03.0/build/dockerfile.3.0.0/dockerfile-7-b38251.err
### stderr ###
# Error: The implementation src/dockerfile.pp.ml
# [...]
#        The value `t_of_sexp' is required but not provided
#        The value `sexp_of_t' is required but not provided
#     ocamlopt src/dockerfile.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.03.0/bin/ocamlopt.opt -w -40 -g -I /home/opam/.opam/4.03.0/lib/fmt -I /home/opam/.opam/4.03.0/lib/ocaml -I /home/opam/.opam/4.03.0/lib/result -I /home/opam/.opam/4.03.0/lib/sexplib -I /home/opam/.opam/4.03.0/lib/uchar -no-alias-deps -I src -o src/dockerfile.cmx -c -impl src/dockerfile.pp.ml)
# File "src/dockerfile.ml", line 1:
# Error: The implementation src/dockerfile.pp.ml
#        does not match the interface src/dockerfile.cmi:
#        The value `t_of_sexp' is required but not provided
#        The value `sexp_of_t' is required but not provided
```

**Comment by smondet** [comment]
I see that the opam-file doesn't say anything about sexplib versions, so maybe a constraint is missing?

My build picked-up:

```
  - install ppx_driver              113.33.02+4.03
  - install ppx_fields_conv         113.33.00+4.03
  - install ppx_optcomp             113.33.00+4.03
  - install ppx_sexp_conv           113.33.01+4.03
  - install ppx_tools               5.0+4.03.0    
  - install ppx_tools_versioned     5.0.1         
  - install ppx_type_conv           113.33.02+4.03
...
  - install sexplib                 113.33.00+4.0
...
```

**Comment by avsm** [comment]
that's right -- fixed in 3.1.1. thanks for the report!
